test/e2e/flowexporter_test.go

```diff
@@ -51,6 +52,56 @@ func TestFlowExporter(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error when getting the perftest server Pod's IP: %v", err)
 	}
+
```

Comment:
maybe it's worth adding a comment somewhere, that because both source & destination Pods run on the same Node, we do not receive duplicate records from 2 different Nodes and records include "complete" information (e.g. both ingress & egress policy information)